src/Integration.Vsix/Analysis/IssueConsumerFactory_IssueHandler.cs

```diff
@@ -99,11 +99,11 @@ internal class IssueHandler
                 onSnapshotChanged(newSnapshot);
             }
 
-            private void MarkSuppressedIssues(IEnumerable<IAnalysisIssueVisualization> issues)
+            private void MatchServerIssues(IEnumerable<IAnalysisIssueVisualization> issues)
```

Comment:
method name should be "Mark" not "Match" (unless the change is intentional)

Reply:
It is intentional. The end goal of this series of PRs is to match the issues instead of just marking them.